packages/office-addin-debugging/src/start.ts

```diff
@@ -8,10 +8,13 @@ import * as devSettings from "office-addin-dev-settings";
 import { DebuggingMethod } from "office-addin-dev-settings";
 import * as manifest from "office-addin-manifest";
 import * as nodeDebugger from "office-addin-node-debugger";
+import * as os from "os";
+import * as path from "path";
 import { getProcessIdsForPort } from "./port";
 import { startDetachedProcess, startProcess  } from "./process";
 
-export const processIdFile = "devServerProcessId.txt";
+const processIdFile = "devServerProcessId.txt";
```

Comment:
Suggest making the name ".office-addin-debugging" and in the file perhaps write it as a JSON object:

```
{
  "devServer": {
    "processId": 123
   }
}
```
That would be more extensible to having other info in there. 

Also it would better reflect that this is related to Office add-in debugging. 

Do you think this would be an easy change? What do you think?

Reply:
The variable names could be updated to reflect it, e.g. debugInfoFile.

Reply:
If you think it's overkill to do as a JSON object, then we could just do the file contents the same way but rename it to reflect that it is for debugging.